### ISSUE TITLE
Paths with spaces not working in config file

### ISSUE THREAD

**Issue by demonbane** [ehendrix23/tesla_dashcam#89]
I'm trying to provide a path for `--font` in the config file, and no matter what combination of `'`, `"`, and `\` I try, paths with spaces never get recognized. It works fine from the command line, but in the config file no luck. Among the variants I've tried:

```
--font /Path/to/file name.ttf
--font /Path/to/file\ name.ttf
--font "/Path/to/file name.ttf"
--font '/Path/to/file name.ttf'
--font "/Path/to/file\ name.ttf"
--font '/Path/to/file\ name.ttf'
--font /Path/to/file\\ name.ttf
--font /Path/to/file\\\ name.ttf
```

**Comment by ehendrix23** [comment]
This is a bug. You can combine command line and config file so for now work-around would be to provide --font on command line.

**Comment by ehendrix23** [comment]
Fixed in release 0.1.16
